src/content/getting-started/architecture/networkplugin-syncer/_index.en.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Networkplugin syncer"
+---
+
+The **_networkplugin-syncer_** interfaces to the K8s network plugin subsystem
+in any necessary way. Currently, it's only deployed for _OVNKubernetes_,
```

Comment:
For all the other components like (Gateway Engine, Route Agent etc) we are not using bold/italic representation. Can we keep the same even for networkplugin-syncer.